app/main.py

```diff
@@ -1,6 +1,31 @@
-from typing import Callable
+from typing import Callable, Any
+from functools import wraps
+
+
+def clear_cache() -> None:
+    cached_data.clear()
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    @wraps(func)
```

Comment:
You should create cached_data dict in `cache` function. That's how you'll use closure in this case